### ISSUE TITLE
Laravel zero - Use external environment in build app

### ISSUE THREAD

**Issue by kaleb14587** [laravel-zero/laravel-zero#191]
I use the .env normally but when I do the application build I would like to use another production .env and I just can not do it.

is it possible, how do I do this?

**Comment by nunomaduro** [comment]
cc @ijpatricio

**Comment by ijpatricio** [comment]
Hello guys,

@nunomaduro ok, as previewed in https://github.com/laravel-zero/laravel-zero/issues/183#issuecomment-463695198

We can make this with external `.env` file, living aside to the built app.
So, its the conventioned KEY=VALUE pairs, no support for arrays, like in a `.php` file

Like this, right?
```
.
├── .env
└── laravel-zero-app
```

Should the name be the conventioned `.env` hidden file?
Maybe this filename be a configuration?
What are your thoughts on this?

Thanks

**Comment by kaleb14587** [comment]
Hy, 
I want the build file to use an env to file not that it builds with env.  
Something like:
```
├─ application.php
├─ .env
└── builds  
       └── application (binary)
       ├── .production.env
```

**Comment by kaleb14587** [comment]
I try do it in appserviceprovider but the database connection  has established on function boot() of the provider.
```
if (app()->environment() == "production"  
 and env('ENV_LOCAL') == "development" and file_exists(".environment") ) {
      $envFile = app()->environmentFilePath();
      $str = file_get_contents($envFile);
      $envprod = $this->envToArray(file_get_contents(".environment"));
      $env = $this->envToArray($str);

      $env = $envprod + $env;

      foreach ($env as $setting) {
        foreach ($setting as $key => $value) {
          if (!empty($key)) putenv($key . "=" . $value);
        }
      }
      DB::connection('cluster21')->reconnect();
      DB::connection('bitnami_drupal8')->reconnect();
    }
```

**Comment by kaleb14587** [comment]
Great,  
I solved with #183 comment
